docs/moderator.md

```diff
@@ -0,0 +1,86 @@
+> Hello!
+>
+> Welcome to the Angular Community team! We're glad you're here. You're helping
+> ensure the community stays welcoming for everyone. Below, you will find an
+> overview of the server and a frequently asked questions of the most common
+> scenarios you will run into.
+>
+> If you have any questions about the server or need help with anything, please
+> do not hesitate to reach out to myself or to another `@Staff` member.
+>
+> Sincerely,
+>
+> delasteve
+
+## Preface
+
+Before we begin, you'll need to enable the `Developer Mode` feature in Discord. You
+can find how to do that [here][discord-developer-mode]. This will help ensure
+you're able to view user ids in the event you have to take action against a member
+who violates our [Code of Conduct (CoC)](../CODE_OF_CONDUCT.md) or
+[server rules](../README.md). To find the id of a user, simply right click their
+username and select `Copy ID` from the menu that appears. This also works for just
+about everything, such as messages, channels, and servers.
+
+## Server Overview
+
+Upon getting the `@Moderator` role, the first thing you'll notice is many new
+channels. These are set up for logging purposes. Hopefully, we will never need
```

Comment:
`the first thing you will notice is a new category, called logs, with quite a lot of channels. As the name implies, all of these channels are for logging purposes. Hopefully, we will never need to use them, but they will be invaluable when we do.`